src/ray/core_worker/core_worker.cc

```diff
@@ -212,9 +212,9 @@ CoreWorker::CoreWorker(const WorkerType worker_type, const Language language,
     SetCurrentTaskId(task_id);
   }
 
-  auto client_factory = [this](const rpc::WorkerAddress &addr) {
+  auto client_factory = [this](const std::string ip_address, int port) {
```

Comment:
Make addr/port explicit here since WorkerAddress now contains 4 fields instead of just the 2 needed.